test/validation/api/Makefile.am

```diff
@@ -56,6 +56,7 @@ TESTS = \
 	ipsec/ipsec_inline_out$(EXEEXT) \
 	lock/lock_main$(EXEEXT) \
 	packet/packet_main$(EXEEXT) \
+	pktio/pktio_main$(EXEEXT) \
```

Comment:
This test will default to using "loop" interface. So in case of linux-generic you could remove the now unnecessary loop pktio test in `platform/linux-generic/test/validation/api/pktio/pktio_run.sh`.